kit/src/main/java/com/oracle/javafx/scenebuilder/kit/editor/EditorPlatform.java

```diff
@@ -89,17 +89,17 @@ public static OS get() {
     /**
      * This URL is where you go when the user takes Scene Builder Help action (shortcut F1)
      */
-    public static final String DOCUMENTATION_URL = "https://docs.oracle.com/javafx/index.html"; //NOI18N
+    public static final String DOCUMENTATION_URL = DocumentationUrls.getInstance().getOracleDocumentation();
```

Comment:
Can we generalise this to the new addition in this PR, in which case these 3 public static finals can go?

Reply:
Yes.

Reply:
These 3 instances have been removed. The places where those were used have been updated accordingly.